packages/swc-helpers/src/_class_private_field_update.js

```diff
@@ -0,0 +1,7 @@
+import classExtractFieldDescriptor from "./_class_extract_field_descriptor";
```

Comment:
Is this naming intentioal?

Reply:
Do you mean the prefix `_` from `_classApplyDescriptorUpdate`?
it's a typo.